### ISSUE TITLE
Question: InfiniteBoost vs XGBoost ?

### ISSUE THREAD

**Issue by vfdev-5** [arogozhnikov/infiniteboost#2]
This is more a question than an issue (I can close it at any time), did you compare your approach with XGBoost implementation ? It could be interesting to compare, especially on overfitting. 

A small typo [here](https://github.com/arogozhnikov/infiniteboost/blame/master/README.md#L54) : `InfiniteBost -> InfiniteBoost`

Thanks

**Comment by arogozhnikov** [comment]
Hi, 
first, difference: XGBoost is mostly a very successful implementation of gradient boosting, while InfiniteBoost is a modification of the standard contribution encountering scheme, that presents inside all implementations (at least, all implementations I am aware of), including XGBoost. 

Implementation that was used for tests contains all major details of XGBoost (those mentioned in [XGBoost paper](https://arxiv.org/abs/1603.02754)), but this implementation is simplistic and lacks many additional parameters related to trees that XGBoost has. So in some sense we have compared to particular case of XGBoost, while some minor details may be different. At least, I am sure that you'll observe the same overfitting patterns with XGBoost as we show in the paper.

**Comment by vfdev-5** [comment]
Thanks
